FlightReservation/src/consts/style.js

```diff
@@ -274,6 +277,43 @@ const style = StyleSheet.create({
   // IMAGES
   imgFlight: {width: 400, height: 450, tintColor: colors.green, opacity: 0.4},
   imgCode: {width: 360, height: 200, tintColor: colors.dark, opacity: 0.8},
+
+  //* Spinner Modal *//
+  center_item: {
+    flexDirection: 'column',
+    justifyContent: 'center',
+    alignItems: 'center',
+  },
+  modal_container: {
+    backgroundColor: 'rgba(0,0,0,0.5)',
```

Comment:
You can substitute that rgb, by the hexadecimal value (called from your color library):

```suggestion
    backgroundColor: 'white',
    opacity: 0.5,
```
You could also try:

```suggestion
    backgroundColor: 'transparent',
```
Or through the Hexadecimal Alpha:

```suggestion
    backgroundColor: '#00000000',
```

Reply:
We know there are several ways to do a same thing, nevertheless this is a container and doing the way you are suggesting will add opacity to child elements. That's why we add style to that specific element using this line.